Localization/StringsConvertor/Sources/StringsConvertor/main.swift

```diff
@@ -74,6 +74,7 @@ private func map(language: String) -> String? {
     case "th.lproj":        return "th"         // Thai
     case "tr.lproj":        return "tr"         // Turkish
     case "vi.lproj":        return "vi"         // Vietnamese
+    case "el.lproj":        return "el"         // Vietnamese
```

Comment:
💡 Let's update the comment here as well

Reply:
Oh, that's slightly embarrassing 🙈 Good, that you saw that!